src/html/contact.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
Please refactor all HTML in to React components. There should not be any new `.html` files.